### DISCUSSION TITLE
Amazon Prime USA only

### DISCUSSION THREAD

**Discussion by SegAgeS** [trakt-tools/universal-trakt-scrobbler#184] [Q&A]
I use amazon prime in the uk - .co.uk however the scrobbler only reads .com (usa).

Is there a way for this to read uk amazon?

**Comment by dafunks** [comment]
Was hoping something would have been done. Any updates on this?

**Comment by Ted011111** [comment]
Same here

**Comment by TDanks2000** [comment]
Same here

**Comment by dafunks** [comment]
Just checking in to show interest in this again. I hope it happens
